infra/blueprint-test/pkg/discovery/discover.go

```diff
@@ -69,21 +69,31 @@ func FindTestConfigs(t testing.TB, intTestDir string) []string {
 	if err != nil {
 		t.Logf("Error discovering examples: %v", err)
 	}
-	testCases := make([]string, 0)
-	//TODO(bharathkkb): add overrides
+	testCases := make(map[string]string)
+	testConfig, err := getTestConfig(defaultTestConfigFilename)
```

Comment:
I was thinking each blueprint test directory would have its own config. What do you think of that (ex. `examples/my-test/test.yaml`)?

Especially for mono-repos. that seems better than collapsing all config into one file.

Reply:
That sounds good to me. Does this API per test seem reasonable to start with?
```yaml
apiVersion: blueprints.cloud.google.com/v1alpha1
kind: BlueprintTest
metadata:
  name: test
spec:
  skip: true
```

Reply:
Yep, LGTM.

Reply:
done